### PR TITLE
Fix crash when trying to bisect a batch

### PR DESCRIPTION
**Pull request by notriddle**
Fixes a crash that happened to badboy, and probably others.

Regression introduced by #121

### CODE REVIEW

File: apps/bors_worker/lib/batcher.ex [hunk 1]
```diff
@@ -344,19 +344,21 @@ defmodule BorsNG.Worker.Batcher do
     project = batch.project
     repo_conn = get_repo_conn(project)
     erred = Enum.filter(statuses, &(&1.state == Status.numberize_state(:error)))
-    patches = batch.id
-    |> Patch.all_for_batch()
+    patch_links = batch.id
+    |> LinkPatchBatch.from_batch()
     |> Repo.all()
-    state = bisect(patches, batch)
+    patches = Enum.map(patch_links, &(&1.patch))
```
Comment (Macarse) [inline apps/bors_worker/lib/batcher.ex]: `&(&1.patch)`? what's that?

Comment (notriddle) [inline apps/bors_worker/lib/batcher.ex]: Shorthand for `fn link -> link.patch end`

It's an anonymous function.

**Comment by notriddle** [comment]
Dogma was complaining.

bors r=Macarse

**Pull request merged**